packages/icons/README.md

```diff
@@ -1,6 +1,6 @@
-# Scalar Icon Library
+# Scalar Icons
 
-The Scalar Icon Library
+All the icons we’re using
```

Comment:
This is actually an icon library that we use to provide user selectable icons, it's distinct from `@scalar/components → ScalarIcon` which uses a different list of more UI focused icons.